lib/commands/actions.js

```diff
@@ -312,6 +312,28 @@ commands.networkSpeed = async function networkSpeed (networkSpeed) {
   await this.adb.networkSpeed(networkSpeed);
 };
 
+/**
+ * Emulate sensors values on the connected emulator.
+ *
+ * @param {object} opts - Object containing sensor type declared in adb.SENSORS items and value
```

Comment:
Please use appropriate format to describe this type. Check https://github.com/appium/appium-adb/blob/c00e48e9977f024ca3280e1ecf6b823997594779/lib/tools/system-calls.js#L141 as an example